c7n/resources/awslambda.py

```diff
@@ -257,6 +258,28 @@ def process(self, resources, event=None):
             resources, event)
 
 
+@AWSLambda.filter_registry.register('kms-key')
+class KmsFilter(KmsRelatedFilter):
+    """
+    Filter a resource by its associcated kms key and optionally the aliasname
+    of the kms key by using 'c7n:AliasName'
+
+    :example:
+
+        .. code-block:: yaml
+
+            policies:
+                - name: dynamodb-kms-key-filters
+                  resource: dynamodb-table
```

Comment:
Might want to fix the doc string with the correct resource type

Reply:
And thats why we do code reviews ;)

Reply:
Updated in 96b3b0f